### ISSUE TITLE
Exercise Answer Key?

### ISSUE THREAD

**Issue by cledoux** [CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers#266]
It would be useful if there existed an answer key for the exercises.

With such a key, the feedback obsessed like myself could verify they aren't making dumb mistakes or misunderstanding the exercise in some way.

**Comment by CamDavidsonPilon** [comment]
Hmm, I thought I did add an answer key. Thanks for the suggestion/reminding me!

**Comment by ClementC** [comment]
+1 for the answer key request.

I started the book and I can already say that the approach you took by providing IPython notebooks is the best I've seen to let people experiment hands-in on a new topic! Thanks a lot!

**Comment by zbessinger** [comment]
+1 for the answer key as well!

**Comment by MLeadbetter** [comment]
Really enjoying it so far. An answer key would be great though.

Cheers!

**Comment by eddiebf** [comment]
+1 for answer key!

**Comment by A-Jacobson** [comment]
I agree the other posters. Loving the book so far, but an answer key would be reassuring. 

Thanks for all your work!

**Comment by kernelmode** [comment]
+1, having the answer key would be really great

**Comment by nigeljyng** [comment]
+1, I agree with the other posters

**Comment by atabakd** [comment]
I +1 too.

**Comment by khuevu** [comment]
Thanks for the book. +1 for this.

**Comment by monapasan** [comment]
any updates on this?

**Comment by mvanlonden** [comment]
the answers are available in the [printed version](https://smile.amazon.com/Bayesian-Methods-Hackers-Probabilistic-Addison-Wesley/dp/0133902838?ie=UTF8&*Version*=1&*entries*=0)

**Comment by hsm207** [comment]
Why not make the answers available online too?
